includes/templates/partials/header-template.php

```diff
@@ -25,35 +25,35 @@
 	<div class="Omise-Box Omise-Account">
 		<dl>
 			<!-- Account email -->
-			<dt>Account: </dt>
+			<dt><?php echo Omise_Util::translate( 'Account' ); ?>: </dt>
 			<dd><?php echo $omise['account']['email']; ?></dd>
 
 			<!-- Account status -->
-			<dt>Mode: </dt>
-			<dd><strong><?php echo $omise['balance']['livemode'] ? '<span class="Omise-LIVEMODE">LIVE</span>' : '<span class="Omise-TESTMODE">TEST</span>'; ?></strong></dd>
+			<dt><?php echo Omise_Util::translate( 'Mode' ); ?>: </dt>
+			<dd><strong><?php echo $omise['balance']['livemode'] ? '<span class="Omise-LIVEMODE">' . Omise_Util::translate( 'LIVE' ) . '</span>' : '<span class="Omise-TESTMODE">' . Omise_Util::translate( 'TEST' ) . '</span>'; ?></strong></dd>
 
 			<!-- Current Currency -->
-			<dt>Currency: </dt>
+			<dt><?php echo Omise_Util::translate( 'Currency' ); ?>: </dt>
 			<dd><?php echo strtoupper( $omise['balance']['currency'] ); ?></dd>
 
 			<!-- Payment Action -->
-			<dt>Auto Capture: </dt>
-			<dd><?php echo $viewData['auto_capture']; ?></dd>
+			<dt><?php echo Omise_Util::translate( 'Auto Capture', 'Account information' ); ?>: </dt>
+			<dd><?php echo $viewData['auto_capture'] == 'YES' ? Omise_Util::translate( 'YES', 'Auto capture status is enabled' ) : Omise_Util::translate( 'NO', 'Auto capture status is disabled' ); ?></dd>
 
 			<!-- 3D Secure enabled? -->
-			<dt>3D-Secure: </dt>
-			<dd><?php echo $viewData['support_3dsecure']; ?></dd>
+			<dt><?php echo Omise_Util::translate( '3-D Secure' ); ?>: </dt>
+			<dd><?php echo $viewData['support_3dsecure'] == 'ENABLED' ? Omise_Util::translate( 'ENABLED', '3-D Secure status is enabled' ) : Omise_Util::translate( 'DISABLED', '3-D Secure status is disabled' ); ?></dd>
```

Comment:
@guzzilar WordPress allows the duplicated context but the PO file was automatically generated by the program, Poedit.

Normally, Poedit can not detect the syntax that the string for translation was replace with variable.